client/my-sites/stats/site.jsx

```diff
@@ -314,6 +348,7 @@ class StatsSite extends Component {
 							period={ this.props.period }
 							chartTab={ this.props.chartTab }
 							customQuantity={ this.state.customChartQuantity }
+							customRange={ customChartRange }
```

Comment:
Does the custom range only have `chartEnd` property?

Reply:
Currently, yes. I was thinking we could put the quantity value inside the chart range as well as the `chartStart` value (when provided). That way we only push a single range-related object around the hierarchy. For example, we need the start and end dates to properly set the button label, which it currently doesn't do.